incubator/hnc/api/v1alpha2/hierarchy_types.go

```diff
@@ -112,7 +112,7 @@ type HierarchyConfigurationSpec struct {
 	Parent string `json:"parent,omitempty"`
 
 	// AllowCascadingDelete indicates if the subnamespaces of this namespace are allowed to cascading
-	// delete.
+	// delete. TODO update it to allowCascadingDeletion.
```

Comment:
FYI This is added just for controller-gen to generate a `per-version` schema for this HC CRD. Since all fields were the same, it still generated the `top-level` schema without this minor difference.